### PR TITLE
Bugfix/djangoenv file not callable from backend

### PR DESCRIPTION
**Pull request by Marcodaum**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,4 @@
 backend/env
 **/__pycache__
 .DS_Store
-**/django.env
+.env
```
Comment (FabianBretz) [inline .gitignore:6]: Does this rule apply for all .env files in any subfolder? Or does it only ignore the .env files in the root directory?

**Review by FabianBretz — CHANGES_REQUESTED** [review]
One quick question

File: .gitignore (hunk 1 above)
Comment (Marcodaum) [reply .gitignore]: I am not so sure about it but I have changed it to **/.env.

**Review by FabianBretz — APPROVED** [review]
lgtm

**Review by 23ari — APPROVED** [review]

**Pull request merged**